### DISCUSSION TITLE
Don't upload and deploy if specific folder changed

### DISCUSSION THREAD

**Discussion by MrCrumbs** [vercel/vercel#8872] [Help] [answered]
Hey,

I have a git repo with 2 folders: "backend" and "frontend". The "backend" folder is the relevant one for Vercel (the "frontend" is on Firebase). 
I do not want any commits to "frontend" to be pushed to Vercel and trigger a deployment. So I tried the following:

* I configured "backend" under "Root Directory". 
* I also de-selected "Include source files outside of the Root Directory in the Build Step". 
* I added `../frontend` to `.vercelignore`.

But still, any commit I make to "frontend" triggers a deploy on Vercel. 
I know I can simply split the repo into two repos. But is there a way to tell Vercel to ignore commits to "frontend"?

**Comment by amyegan** [comment, marked as answer]
Hi @MrCrumbs! Have you seen the [help guide about Ignored Build Step](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel)? There is a section about configuring it for [folders and workspaces](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel#with-folders-and-workspaces) that should help.

_tl;dr_ You should be able to use `git diff HEAD^ HEAD --quiet .` in the "Ignored Build Step" field to ignore all changes outside of the Root Directory.

**Reply by MrCrumbs** [reply]
Perfect, thanks!
